src/test/mochitest/browser_dbg-breakpoints-cond.js

```diff
@@ -39,7 +39,10 @@ async function assertConditionalBreakpointIsFocused(dbg) {
 
 async function setConditionalBreakpoint(dbg, index, condition) {
   rightClickElement(dbg, "gutter", index);
-  selectMenuItem(dbg, 2);
+  selectContextMenuItem(
+    dbg,
+    "#node-menu-add-conditional-breakpoint,#node-menu-edit-conditional-breakpoint"
+  );
```

Comment:
This one is kind of tricky.  When this is called from line 60 the context menu item at index 2 is **add conditional breakpoint**.  When it is called from line 66 the context menu item at index 2 is **edit conditional breakpoint**.  So I used a selector that will get either one of these possibilities.  A better alternative might be making this into separate functions for adding and editing.